lib.go

```diff
@@ -10,6 +10,23 @@ type VM struct {
 	Inner api.VM
 }
 
+type Compiler struct {
+	Inner api.Compiler
+}
+
+func NewCompiler() Compiler {
+	inner := api.InitCompiler()
+	return Compiler{inner}
+}
+
+func (compiler *Compiler) Start(kvstore api.KVStore) {
+	api.StartRoutine(compiler.Inner, kvstore)
+}
+
+func (compiler *Compiler) Destroy() {
+	api.ReleaseCompiler(compiler.Inner)
+}
+
```

Comment:
use same db(kvstore) of vm store not the copy of db(kvstore). if vm store db is commited by tx, but compiler kvstore db is still not changed.